src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java

```diff
@@ -18,4 +25,16 @@ public class RespondentLiPResponse {
     private Address respondent1LiPCorrespondenceAddress;
     private DQExtraDetailsLip respondent1DQExtraDetails;
     private HearingSupportLip respondent1DQHearingSupportLip;
+    private String respondent1ResponseLanguage;
+
+    @JsonIgnore
+    public boolean isRespondentResponseBilingual(CaseData caseData) {
+        boolean responseLanguage = Optional.ofNullable(caseData.getCaseDataLiP())
+            .map(CaseDataLiP::getRespondent1LiPResponse)
+            .map(RespondentLiPResponse::getRespondent1ResponseLanguage)
+            .filter(respondent1ResponseLanguage -> Language.BOTH.toString().equals(respondent1ResponseLanguage))
```

Comment:
sorry in my preview comment I make a mistake...sorry :(
this line could be         .filter(Language.BOTH.toString()::equals)

Reply:
my mistake.. didnt realized it. Thanks for pointing it.